package.json

```
@@ -37,6 +37,7 @@
     "react-dom": "~16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz",
     "react-native-aws3": "0.0.9",
+    "react-native-datepicker": "^1.7.2",
```

```suggestion
    "react-native-datepicker": "1.7.2",
```

So we lock the dependencies